### DISCUSSION TITLE
expo run:ios --device doesn't show my connected device

### DISCUSSION THREAD

**Discussion by Akumzy** [expo/expo-cli#4217] [Q&A] [answered]
I recently changed my MacBook pro and since then I haven't been lucky to connect to my iPhone 12 as usual with `expo run:ios --device`

It's only showing my simulators, what could I be missing?

**Comment by kbrandwijk** [comment]
Does your device show up in XCode?

**Reply by Akumzy** [reply]
Yes I do, secondly I updated to iOS 15.4 I don't know if that has any relationship  but it was working on my old Macbook

**Comment by Akumzy** [comment, marked as answer]
Issue was caused by my Xcode not supporting the iOS version my phone was running, I installed the beta version of Xcode before it could work as I can't find an easy way to downgrade my iOS version
